package.json

```diff
@@ -29,10 +29,10 @@
 		"reflect-metadata": "^0.1.13",
 		"retire": "^2.2.5",
 		"semver": "^7.3.4",
-		"ts-node": "^8",
+		"ts-node": "^10",
 		"tslib": "^2",
 		"tsyringe": "^4.1.0",
-		"typescript": "^3.8.2",
+		"typescript": "^4.6.2",
```

Comment:
3pp for typescript: https://gus.lightning.force.com/lightning/r/a0qEE00000023zVYAQ/view